airbyte-analytics/src/main/java/io/airbyte/analytics/TrackingClientSingleton.java

```diff
@@ -70,7 +70,7 @@ private static void initialize() {
   @VisibleForTesting
   static TrackingIdentity getTrackingIdentity(ConfigRepository configRepository, String airbyteVersion) {
     try {
-      final StandardWorkspace workspace = configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID);
+      final StandardWorkspace workspace = configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID, false);
```

Comment:
to give you a little context, the concept of a "workspace" and a "deployment" are conflated in our system right now. that has been okay so far because there is only one workspace. the reason i mention it, is that we still to be able to use the default workspace for deployment info (even if the default workspace has been deleted).

all that being said, i think that just means we want `true` here instead of false. if the default workspace is ever deleted we still want this method to be able to access it.